docs/changelog/undistributed/changelog_apic_xml_20210729113240.rst

```diff
@@ -0,0 +1,3 @@
+
+* APIC
+  * Added XML payload option to push method
```

Comment:
thanks for this contribution. can I ask you to add unittest for this change to below file?
https://github.com/CiscoTestAutomation/rest/blob/master/src/rest/connector/tests/test_apic.py

Reply:
Added two new tests to test_apic unittest